device-types/Ciena/6500-R2.yaml

```diff
@@ -0,0 +1,35 @@
+---
+manufacturer: Ciena
+model: 6500 R2
+slug: ciena-6500-r2
+part_number: NTK803DA
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it